examples/src/example-udp-input.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * example-udp-input
+ *
+ * This example demonstrates how to use the UDP Core Input
+
+ * DEPENDENCIES:
+ * - log4js - is a generic logger for NodeJS. See the file ./providers/custom_logger.js for a basic
+ *   example of log4js as a replacement to console.log.
+ *
+ */
+
+// Bring in Package Dependencies
+const { default: RulesHarvester, CoreInputUdp } = require('rule-harvester');
+
+// Bring in other Application Specific dependencies
+const RuleOutputProviderFile = require('./providers/rule_output_file');
+const ruleClosures = require('./providers/rule_closures');
+const ruleCorpus = require('./providers/rule_corpus_udp');
+const logger = require('./providers/custom_logger');
+
+let udpFileCounter = 0;
+
+const coreUdpInput = new CoreInputUdp([3333], logger, {
+  inputContextCallback: (msg) => {
+    // In this example, this callback receives the full AMQP-Cacoon message object (of type ConsumeMessage).
```

Comment:
@daniellmorris this comment is referencing AMQP-Caccoon but that's not what we're getting here. Can you change the comment to refer to whatever the msg is? Is it a string? a buffer? etc.